tools/integration-test/src/tests/mod.rs

```diff
@@ -43,5 +43,5 @@ pub mod manual;
 #[cfg(any(doc, feature = "example"))]
 pub mod example;
 
-#[cfg(any(doc, feature = "forward-packet"))]
+//#[cfg(any(doc, feature = "forward-packet"))]
```

Comment:
We probably need to uncomment this, otherwise we will be running these tests for all chains defined in the workflow